features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/ApplicationSettings/EditApplication.jsx

```diff
@@ -97,6 +97,10 @@ function Edit(props) {
                         valid.invalid = true;
                     }
                     resolve(valid);
+                }).catch(() => {
+                    valid.error = `${owner} is not a valid Subscriber`;
+                    valid.invalid = true;
+                    resolve(valid);
```

Comment:
Why we resolve in catch and then throws back an error in immediate success call ? 
```javascript
throw valid.error;
```

Reply:
Fixed with commit: https://github.com/wso2/carbon-apimgt/pull/9035/commits/41ad606861ef2c433a74938094cff5eadd46bae0